heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesController.java

```diff
@@ -142,11 +135,13 @@ protected boolean submitTopology(String[] appConfs) {
       try {
         deployContainer(appConf);
       } catch (IOException ioe) {
-        LOG.log(Level.SEVERE, "Problem deploying container with config: " + appConf);
-        return false;
+        String message = (ioe instanceof java.net.ConnectException)
```

Comment:
Instead of using `instanceof`, we should catch the `ConnectException` directly and handle it separately. Something like:
```
try {
...
} catch (ConnectException) {
...
} catch (IOException) {
...
}
```